src/game/logic/scriptengine/scriptengine.cpp

```diff
@@ -13,56 +13,9407 @@
+    m_actionTemplates[ScriptAction::TEAM_ALL_USE_COMMANDBUTTON_ON_NEAREST_OBJECTTYPE].m_uiStrings[0] = " ";
+    m_actionTemplates[ScriptAction::TEAM_ALL_USE_COMMANDBUTTON_ON_NEAREST_OBJECTTYPE].m_uiStrings[1] = " use ";
+    m_actionTemplates[ScriptAction::TEAM_ALL_USE_COMMANDBUTTON_ON_NEAREST_OBJECTTYPE].m_uiStrings[2] =
+        " on nearest object of type ";
+    m_actionTemplates[ScriptAction::TEAM_ALL_USE_COMMANDBUTTON_ON_NEAREST_OBJECTTYPE].m_uiStrings[3] = ".";
+
+    m_actionTemplates[ScriptAction::TEAM_PARTIAL_USE_COMMANDBUTTON].m_internalName = "TEAM_PARTIAL_USE_COMMANDBUTTON";
+    m_actionTemplates[ScriptAction::TEAM_PARTIAL_USE_COMMANDBUTTON].m_uiName =
+        "Team_/ Use command ability -- partial -- self.";
+    m_actionTemplates[ScriptAction::TEAM_PARTIAL_USE_COMMANDBUTTON].m_numParameters = 3;
+    m_actionTemplates[ScriptAction::TEAM_PARTIAL_USE_COMMANDBUTTON].m_parameters[0] = Parameter::REAL;
+    m_actionTemplates[ScriptAction::TEAM_PARTIAL_USE_COMMANDBUTTON].m_parameters[1] = Parameter::TEAM;
+    m_actionTemplates[ScriptAction::TEAM_PARTIAL_USE_COMMANDBUTTON].m_parameters[2] = Parameter::UNK_ABILITY_RELATED;
+    m_actionTemplates[ScriptAction::TEAM_PARTIAL_USE_COMMANDBUTTON].m_numUiStrings = 4;
+    m_actionTemplates[ScriptAction::TEAM_PARTIAL_USE_COMMANDBUTTON].m_uiStrings[0] = " ";
+    m_actionTemplates[ScriptAction::TEAM_PARTIAL_USE_COMMANDBUTTON].m_uiStrings[1] = "% of ";
+    m_actionTemplates[ScriptAction::TEAM_PARTIAL_USE_COMMANDBUTTON].m_uiStrings[2] = " perform ";
+    m_actionTemplates[ScriptAction::TEAM_PARTIAL_USE_COMMANDBUTTON].m_uiStrings[3] = ".";
+
+    m_actionTemplates[ScriptAction::TEAM_CAPTURE_NEAREST_UNOWNED_FACTION_UNIT].m_internalName =
+        "TEAM_CAPTURE_NEAREST_UNOWNED_FACTION_UNIT";
+    m_actionTemplates[ScriptAction::TEAM_CAPTURE_NEAREST_UNOWNED_FACTION_UNIT].m_uiName =
+        "Team_/ Capture unowned faction unit -- nearest.";
+    m_actionTemplates[ScriptAction::TEAM_CAPTURE_NEAREST_UNOWNED_FACTION_UNIT].m_numParameters = 1;
+    m_actionTemplates[ScriptAction::TEAM_CAPTURE_NEAREST_UNOWNED_FACTION_UNIT].m_parameters[0] = Parameter::TEAM;
+    m_actionTemplates[ScriptAction::TEAM_CAPTURE_NEAREST_UNOWNED_FACTION_UNIT].m_numUiStrings = 2;
+    m_actionTemplates[ScriptAction::TEAM_CAPTURE_NEAREST_UNOWNED_FACTION_UNIT].m_uiStrings[0] = " ";
+    m_actionTemplates[ScriptAction::TEAM_CAPTURE_NEAREST_UNOWNED_FACTION_UNIT].m_uiStrings[1] =
+        " capture the nearest unowned faction unit.";
+
+    m_actionTemplates[ScriptAction::PLAYER_CREATE_TEAM_FROM_CAPTURED_UNITS].m_internalName =
+        "PLAYER_CREATE_TEAM_FROM_CAPTURED_UNITS";
+    m_actionTemplates[ScriptAction::PLAYER_CREATE_TEAM_FROM_CAPTURED_UNITS].m_uiName =
+        "Player_/ Create team from all captured units.";
+    m_actionTemplates[ScriptAction::PLAYER_CREATE_TEAM_FROM_CAPTURED_UNITS].m_numParameters = 2;
+    m_actionTemplates[ScriptAction::PLAYER_CREATE_TEAM_FROM_C
```

Comment:
Text accommodates third parameter, but only 2 given. `m_parameters[2] = Parameter::SURFACES_ALLOWED;` missing?

Reply:
This is correct.